### PR TITLE
block detail display on react

### PR DESCRIPTION
**Pull request by takafumi-hino**
created block detail display  working on reactjs.

public dir is not changed.
programs in the src dir.

please check my programs.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+table , td, th {
```
Comment (rantan) [inline src/index.css]: It seems that these styles are not used anywhere. Can you eliminate this?

File: src/server.js [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/hino/server/server.js
```
Comment (rantan) [inline src/server.js]: What is this change?

File: src/index.css (hunk 1 above)
Comment (takafumi-hino) [reply src/index.css]: These styles are used by index.js for displaying block list.
So it cannot be deleted.

File: src/server.js (hunk 2 above)
Comment (takafumi-hino) [reply src/server.js]: No change anywhere.
I just applied the previous commit comment.

**Comment by rantan** [comment]
close this PR because this changes are included in PR #7 .

**Pull request closed without merging**